napari/_vispy/layers/points.py

```diff
@@ -188,9 +189,8 @@ def _on_canvas_size_limits_change(self):
     def reset(self):
         super().reset()
         self._update_text(update_node=False)
-        self._on_blending_change()
+        self._on_symbol_change()
```

Comment:
```suggestion
        self._on_blending_change()
        self._on_symbol_change()
```

Reply:
Forgot to mention that in the description above, but that is also already present in the base layer, so I removed it on purpose! Yup, the print snuck in xD

Reply:
okay cool, in that case LGTM!